trade-finance-manager-api/src/v1/controllers/update-issued-facilities.js

```diff
@@ -21,19 +21,27 @@ const updatedIssuedFacilities = async (deal) => {
 
   modifiedDeal.facilities = await Promise.all(modifiedDeal.facilities.map(async (f) => {
     const facility = f;
-
     const {
       _id: facilityId,
       hasBeenIssued,
       hasBeenIssuedAndAcknowledged,
     } = facility;
 
     /**
-     * if hasBeenIssued and hasBeenIssuedAndAcknowledged
-     * ensures tasks only done once and email only sent once for each issued facility
-     * hasBeenIssuedAndAcknowledged only set by tfm-api during this step
+     * `hasBeenIssued` : Facility has been issued by the maker.
+     * `sendToUkef` : Facility has been sent to UKEF.
+     *
+     * Ensures tasks only done once and email only sent once for each issued
+     * facility hasBeenIssuedAndAcknowledged only set by tfm-api during this step.
+     *
+     * If MIN then set `sendToUkef` to `false` to accommodate any cover start date
+     * amendment.
     */
-    if (hasBeenIssued && !hasBeenIssuedAndAcknowledged) {
+    const sendToUkef = submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIN
```

Comment:
based on this comment
> * `sendToUkef` : Facility has been sent to UKEF.

should this be `sentToUkef`?

Reply:
Good spot, have updated.